mitxpro/settings.py

```diff
@@ -1432,4 +1432,10 @@
     description="Comma separated string of course/program runs/Ids that support digital credentials",
 )
 
+ENABLE_COURSE_DROPDOWN = get_bool(
```

Comment:
The standard name format to define feature flags is `FEATURE_*`. Which in this case could be `FEATURE_COURSE_DROPDOWN`.

The reason for this is that the manipulation of the setting from the environment file automatically handles the features and adds them in `setting.FEATURES` dict which we can then use anywhere. An example of adding that can be seen in https://github.com/mitodl/mitxpro/commit/0ade3634608a9b65f83ccc6ebc3faa6c403de70b